packages/wrangler/src/dialogs.tsx

```diff
@@ -83,3 +83,9 @@ export async function prompt(text: string, type: "text" | "password" = "text") {
 // ): Promise<{ label: string; value: string }>{
 
 // }
+
+// Export as an object so that it is easier to mock for testing
```

Comment:
This shouldn't be necessary, but I'll swing by next week and figure out mocking that'll let us keep the original exports. (Dropping this comment here to remind myself)

Reply:
I tried a number of different ways to achieve this by "mocking" the module. But it either didn't work or was really messy. I would be interested to see your approach.